MeesGame/MeesGame.csproj

```diff
@@ -160,6 +160,9 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="Gameplay\Content\button_click.wav" />
+    <Content Include="Gameplay\Content\climbing_sound.wav">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Don't think this line should be here as it isn't there for any of the other files, if it's intentional and for a good reason let me know though.